### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yfarjoun**
add emoticons

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
+![Fulcrum Genomics logo](https://avatars.githubusercontent.com/u/13546013?s=200&v=4)
```
Comment (nh13) [inline profile/README.md]: 1. Lets shrink the logo using some html (see suggestion)
2. Lets add our full logo (with text).  Can you add it to the repo under `profile/assets/logo.svg`?  You can use the one here: https://github.com/nf-core/website/blob/main/src/assets/images/blog/fastquorum/fulcrumgenomics.svg

```suggestion
<a href="https://fulcrumgenomics.com/" alt="Fulcrum Genomics Logo">
<img src="https://avatars.githubusercontent.com/u/13546013?s=200&v=4" width="64">
</a>
```

**Review by nh13 — CHANGES_REQUESTED** [review]
I am not 100% in love with shoving our logo above "Hi there" as it takes a lot of vertical space, so lets see if what it looks like if we add in our full logo, and reduce the height

File: profile/README.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
+<a href="https://fulcrumgenomics.com/" alt="Fulcrum Genomics Logo">
+<img src="./assets/logo.png" width="64">
```
Comment (nh13) [inline profile/README.md]: It’s a svg not a png

**Comment by yfarjoun** [comment]
ok. fixed.... 😓

**Review by nh13 — CHANGES_REQUESTED** [review]
In [this comment](https://github.com/fulcrumgenomics/.github/pull/3#discussion_r1683012044) I suggested that we use the logo _with text_.  Any chance you tried it?  If so, did it not look better?

File: profile/README.md (hunk 1 above)
Comment (yfarjoun) [reply profile/README.md]: sorry, I misunderstood your request. I'm on it!

**Comment by yfarjoun** [comment]
better now?

**Review by nh13 — APPROVED** [review]
I am now in love with this PR

**Pull request merged**